mafiasi/vault/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+
+from . import views
+
+urlpatterns = [
+    path("", views.IndexView.as_view(), name="vault-index"),
```

Comment:
```suggestion
    path("", views.IndexView.as_view(), name="vault_index"),
```
Names should be snake_case like in the rest of the dashboard.